src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -1242,8 +1248,31 @@ def remove_temp_test_dir(self):
             errors.append("Error removing temporary test files")
         return errors
 
+    def fail(self):
```

Comment:
(pylint-arguments-differ) Number of parameters was 2 in 'Test.fail' and is now 1 in overridden 'TestWithServers.fail' method